import-export-cli/integration/apim/throttlePolicyDTO.go

```diff
@@ -0,0 +1,171 @@
+* under the License.
+ */
+
+package apim
+
+// ApplicationThrottlePolicy : Application Throttle Policy DTO
+type ApplicationThrottlePolicy struct {
+	PolicyName   string `json:"policyName"`
+	DisplayName  string `json:"displayName"`
+	Description  string `json:"description"`
+	IsDeployed   bool   `json:"isDeployed"`
+	Type         string `json:"type"`
+	DefaultLimit struct {
+		Type         string `json:"type"`
+		RequestCount struct {
+			TimeUnit     string `json:"timeUnit"`
+			UnitTime     int    `json:"unitTime"`
+			RequestCount int    `json:"requestCount"`
+		} `json:"requestCount"`
+		Bandwidth struct {
+			TimeUnit   string `json:"timeUnit"`
+			UnitTime   int    `json:"unitTime"`
+			DataAmount int    `json:"dataAmount"`
+			DataUnit   string `json:"dataUnit"`
+		} `json:"bandwidth"`
+		EventCount struct {
+			TimeUnit   string `json:"timeUnit"`
+			UnitTime   int    `json:"unitTime"`
+			EventCount int    `json:"eventCount"`
+		} `json:"eventCount"`
+	} `json:"defaultLimit"`
+}
+
+// CustomThrottlePolicy : Custom Throttle Policy DTO
+type CustomThrottlePolicy struct {
+	PolicyName  string `json:"policyName"`
+	DisplayName string `json:"displayName"`
+	Description string `json:"description"`
+	IsDeployed  bool   `json:"isDeployed"`
+	Type        string `json:"type"`
+	SiddhiQuery string `json:"siddhiQuery"`
+	KeyTemplate string `json:"keyTemplate"`
+}
+
+// AdvancedThrottlePolicy : Advanced Throttle Policy DTO
+type AdvancedThrottlePolicy struct {
+	PolicyName   string `json:"policyName"`
+	DisplayName  string `json:"displayName"`
+	Description  string `json:"description"`
+	IsDeployed   bool   `json:"isDeployed"`
+	Type         string `json:"type"`
+	DefaultLimit struct {
+		Type         string `json:"type"`
+		RequestCount struct {
+			TimeUnit     string `json:"timeUnit"`
+			UnitTime     int    `json:"unitTime"`
+			RequestCount int    `json:"requestCount"`
+		} `json:"requestCount"`
+		Bandwidth struct {
+			TimeUnit   string `json:"timeUnit"`
+			UnitTime   int    `json:"unitTime"`
+			DataAmount int    `json:"dataAmount"`
+			DataUnit   string `json:"dataUnit"`
+		} `json:"bandwidth"`
+		EventCount struct {
+			TimeUnit   string `json:"timeUnit"`
+			UnitTime   int    `json:"unitTime"`
+			EventCount int    `json:"eventCount"`
+		} `json:"eventCount"`
+	} `json:"defaultLimit"`
+	ConditionalGroups []AdvancedPolicyConditionalGroup `json:"conditionalGroups"`
+}
+
+// AdvancedPolicyConditionalGroup : Conditional Groups in Advanced Throttle Policy DTO
+type AdvancedPolicyConditionalGroup struct {
+	Description string                    `json:"description"`
+	Conditions  []AdvancedPolicyCondition `json:"conditions"`
+	Limit       struct {
+		Type         string `json:"type"`
+		RequestCount struct {
+			TimeUnit     string `json:"timeUnit"`
+			UnitTime     int    `json:"unitTime"`
+			RequestCount int    `json:"requestCount"`
+		} `json:"requestCount"`
+		Bandwidth  interface{} `json:"bandwidth"`
```

Comment:
Is this Bandwidth struct similar to the one inside SubscriptionThrottlePolicy? If so, shall we create a separate struct as Bandwidth and reuse it in both places?

Reply:
Yes, It's the same struct. Fixed with 680508410867201cf5f4a96486d372c218b1c522